### PR TITLE
Ansible2.4

### PR DESCRIPTION
**Pull request by durbz**

### CODE REVIEW

File: ansible/roles/common/vars/main.yml [hunk 1]
```diff
@@ -1,2 +1,3 @@
 ---
 ignore_hostnames: [ localhost, 127.0.0.1 ]
+required_ansible_version: 2.4.2.0
```
Comment (jaydg) [inline ansible/roles/common/vars/main.yml]: missing trailing newline

File: packer/scripts/prepare-offline-installation.sh [hunk 2]
```diff
@@ -30,6 +30,9 @@ socat
 patch
 python-backports.ssl-match-hostname
 python-docker
+python-setuptools
+python-pip
+python-wheel
```
Comment (jaydg) [inline packer/scripts/prepare-offline-installation.sh]: These shouldn't be required, as they are present in debian.cfg?

Comment (durbz) [reply packer/scripts/prepare-offline-installation.sh]: Good point, I will update this now.

File: ansible/roles/common/vars/main.yml (hunk 1 above)
Comment (durbz) [reply ansible/roles/common/vars/main.yml]: Will push a fix shortly.

File: ansible/playbooks/procedures/upgrade-ansible.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+- hosts: all
+  tasks:
+   
+  - include_vars: "/home/teamwire/platform/ansible/roles/common/vars/main.yml"
+  - include_vars: "/home/teamwire/platform/ansible/group_vars/all"
```
Comment (jaydg) [inline ansible/playbooks/procedures/upgrade-ansible.yml]: Please use relative paths

File: ansible/roles/common/tasks/main.yml [hunk 4]
```diff
@@ -1,4 +1,13 @@
 ---
+- name: Check Ansible version
+  fail:
+    msg:
+      - "Ansible {{ required_ansible_version }} is required, please run the upgrade procedure"
+      - "First you must change directory to /home/teamwire/platform/ansible"
```
Comment (jaydg) [inline ansible/roles/common/tasks/main.yml]: Please omit this line - if this message is shown, the user is already in the correct directory

File: ansible/roles/common/vars/main.yml (hunk 1 above)
Comment (durbz) [reply ansible/roles/common/vars/main.yml]: New line added and pushed.

File: packer/scripts/prepare-offline-installation.sh (hunk 2 above)
Comment (durbz) [reply packer/scripts/prepare-offline-installation.sh]: I've now removed these unneeded lines.

File: ansible/playbooks/procedures/upgrade-ansible.yml (hunk 3 above)
Comment (durbz) [reply ansible/playbooks/procedures/upgrade-ansible.yml]: I've now set to use relative paths.

File: ansible/roles/common/tasks/main.yml (hunk 4 above)
Comment (durbz) [reply ansible/roles/common/tasks/main.yml]: Removed!

**Pull request merged**